vagrant/scripts/perf-test.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+set -e
+
+# Exemplar script output
+#
+# Starting perf tests
+# Setting up avalanche...
+# Running 1 avalanche(s) with the following arguments:
+# --metric-count=200
+# --series-count=200
+# --port=9006
+# --series-interval=60000
+# --metric-interval=60000
+# --value-interval=30
+#
+# Letting prometheus ingest metrics for 20s...
+# Cleaning up avalanche...
+#
+# Prometheus container memory RSS:
+#         start:           657Mi
+#         finish:         1200Mi
+#         diff:            543Mi
+# receiver-mock ingested 3575 metrics during the test
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+AVALANCHE_YAML_PATH="${DIR}/../k8s/avalanche.yaml"
+readonly RECEIVER_MOCK_POD="$(kubectl get pod -n receiver-mock --no-headers -lapp=receiver-mock --output custom-columns=NAME:.metadata.name)"
```

Comment:
what if there will be more than one receiver-mock pod

Reply:
added `| head -1` :)

Reply:
Thats fine for executing but not for counting the total metrics 😅